Gruntfile.js

```diff
@@ -522,9 +570,27 @@ module.exports = (grunt) => {
     grunt.task.run(`_pin:${version}`)
   })
 
+  grunt.registerTask('deploy', function () {
+    if (!production) {
+      console.log(chalk.yellow('The command has some requirements in setting environment variables.\nNODE_ENV=production'))
```

Comment:
Is this necessary? I can think of some situations where running the debug build would be useful (it's fine to give a warning IMO, just not the `process.exit(1)` that follows)